package.json

```diff
@@ -81,6 +81,7 @@
   "dependencies": {
     "@remixproject/engine": "^0.2.3",
     "http-server": "^0.11.1",
+    "selenium": "^2.20.0",
```

Comment:
not sure you need that here

Reply:
Something went wrong while I was trying to test locally :).  Will remove this. Thanks